src/use-swr.ts

```diff
@@ -45,22 +47,23 @@ const getGlobalState = (cache: Cache) => {
   ]
 }
 
-// Setup DOM events listeners for `focus` and `reconnect` actions
-if (!IS_SERVER) {
-  const [FOCUS_REVALIDATORS, RECONNECT_REVALIDATORS] = getGlobalState(
-    defaultConfig.cache
-  )
+function setupGlobalEvents(cache: Cache, _provider: Partial<Provider> = {}) {
```

Comment:
It seems that we are not using the second argument, what will the use case be link?